msprime/stats.py

```diff
@@ -355,10 +358,9 @@ def Y1_vector(self, sample_sets, windows):
         """
         Finds the 'Y1' statistic within each set of samples in sample_sets. The
         sample_sets should be disjoint (the computation works fine, but if not
```

Comment:
Hm, yes - now as defined `Y1(A) = 0`.  Just as `f1(A) = 0`.  We could define `y1(A)` but I might vote to leave this one out.